frontend/apps/filemanager/filemanager.lua

```diff
@@ -797,7 +803,7 @@ function FileManager:tapPlus()
     UIManager:show(self.file_dialog)
 end
 
-function FileManager:reinit(path, focused_file)
+function FileManager:reinit(path, focused_file, callback)
```

Comment:
Not fond of seeing this additional parameter because of this "little" feature :/
At lease, give it a less anonymous and really explicite name so we can ignore it when reading this code.
`filemanagerdocument_handle_callback` ?